### ISSUE TITLE
Expose IOManager parameter options

### ISSUE THREAD

**Issue by pieper** [Slicer/Slicer#5882]
Parameters for `qSlicerFileReader` and `qSlicerFileWriter` subclasses are hard coded and difficult to discover.

Some of this can be handled by documentation changes [like this](https://github.com/Slicer/Slicer/pull/5881) but something programmatic would be better.

## Describe the solution you'd like

It would be better if classes could expose these properties so they could be introspected and used to generate better documentation [for `slicer.util` methods with properties arguments](https://slicer.readthedocs.io/en/latest/developer_guide/slicer.html#slicer.util.saveNode).

## Describe alternatives you've considered

Currently I search the source code when I need to know the exact parameter name.

**Comment by lassoan** [comment]
Good idea. We already do this with the 'filename' parameter, for the exact same reason of making the parameter more conveniently accessible.
